test/testdrive/mzcompose.py

```diff
@@ -87,10 +88,19 @@ def workflow_testdrive(c: Composition, parser: WorkflowArgumentParser) -> None:
         else ["--aws-endpoint=http://localstack:4566"]
     )
 
+    shard = os.environ.get("BUILDKITE_PARALLEL_JOB")
```

Comment:
I think this should go in the testdrive section of misc/python/materialize/mzcompose/services.py . Otherwise it will get copy-pasted multiple times going forward in various other mzcompose.py files.

Reply:
Maybe I should go back to the original approach of teaching testdrive to read these vars directly. I'm a bit worried about someone just dropping `parallel: 5` on a test like s3-resumption where it is _very_ important that you run testdrive files in sequence.